examples/wireless_wpa3_personal.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Did you test whether the example actually works? On first sight it looks like it is missing the actual `psk`.

Reply:
when I looked at the profile, I got the following:
```
[root@ibm-p8-kvm-03-guest-02 ~]# cat /etc/sysconfig/network-scripts/ifcfg-wlan0 
ESSID="My WPA3-Personal Network"
KEY_MGMT=SAE
MAC_ADDRESS_RANDOMIZATION=default
TYPE=Wireless
PROXY_METHOD=none
BROWSER_ONLY=no
ETHTOOL_OPTS="-K wlan0"
BOOTPROTO=dhcp
DEFROUTE=yes
IPV4_FAILURE_FATAL=no
IPV6INIT=yes
IPV6_AUTOCONF=yes
IPV6_DEFROUTE=yes
IPV6_FAILURE_FATAL=no
IPV6_ADDR_GEN_MODE=stable-privacy
NAME=wlan0
UUID=c2268f28-9957-42ee-8f4b-3a21d052b2d4
DEVICE=wlan0
ONBOOT=yes
```

Reply:
Hi @liangwen12year, the `psk` is missing. You cannot see it doing `cat /etc/sysconfig/network-scripts/ifcfg-wlan0` because it is a secret. In order to see secrets from NM profiles you can do: `nmcli connection show <con_name> --show-secrets`.

There you should find the following entry:
`802-11-wireless-security.psk:           yourpassword`

Reply:
Thanks @ffmancera , I saw the psk in the connection profile.